react/utils/index.tsx

```diff
@@ -50,7 +91,7 @@ export const getSubscriptionInfo = (
   const subsFrequencyString = numberPeriodRegex.test(subsFrequency)
     ? intl.formatMessage(
         {
-          id: `items.attachments.subscription.frequency.${
+          id: `store/items.attachments.subscription.frequency.${
             (numberPeriodRegex.exec(subsFrequency) || [])[2]
```

Comment:
Won't this break if `numberPeriodRegex.exec(subsFrequency)` fails?
Again, my bad 😅

Reply:
I didn't actually change the logic you wrote! But I agree that this is not the ideal way of doing this. Did you have any reason of doing it as it is or can I change it to avoid the case you've mentioned?

Reply:
I know you didn't hahahah and I don't remember having a specific reason to do it this way, most likely I did not realize that it could break when I wrote it.
Sure! Of course, you can!